source/backend/apimodels/Models/Concepts/Organization/OrganizationModel.cs

```diff
@@ -17,9 +18,34 @@ public class OrganizationModel : BaseConcurrentModel
         public long Id { get; set; }
 
         /// <summary>
-        /// get/set - The organization's disabled status flag.
+        /// get/set - The organization's parent organization id.
         /// </summary>
-        public bool IsDisabled { get; set; }
+        public long? ParentOrganizationId { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's reion code.
+        /// </summary>
+        public short? RegionCode { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's district code.
+        /// </summary>
+        public short? DistrictCode { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's type code.
+        /// </summary>
+        public string OrganizationTypeCode { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's identifier code.
+        /// </summary>
+        public string IdentifierTypeCode { get; set; }
```

Comment:
some of these dont seem to be used. @devinleighsmith @asanchezr do you know id these is legacy code and can be removed?

Reply:
As far as I'm aware we don't use this, but I can't check the prod database at the moment to be 100% sure. Can you ask Doug F. maybe?